### ISSUE TITLE
Analize plugin missing feature

### ISSUE THREAD

**Issue by diablo1281** [mavlink/qgroundcontrol#8454]
Hi,
I have just upgrade to version 4.0.0 of QGCS and notice that MAVLink Inspector and Analize plugin get merged. There is a problem however. You can plot only 4 items at once and can't plot specific item from table variable (like field `voltages' from message `BATTERY_STATUS`). I am developing autopilot software and these features are very helpful.

Any chance to bring at least analize tool back?

**Comment by dogmaphobic** [comment]
Indeed, the current graphing does not handle arrays of values. It would be quite tricky to add that. With that said, the theoretical limit is two separate plots with 6 values each.

**Comment by DonLakeFlyer** [comment]
> not handle arrays of values

Did that work before. Don't remember.

**Comment by dogmaphobic** [comment]
Me either but I guess it did, otherwise this wouldn't be brought up. The vast majority of the cases, each field contains one value. Things like battery (cells), quaternions, etc. have an array of values for a single field. The code doesn't handle that at all.

**Comment by DonLakeFlyer** [comment]
Isn't it just another level to the hierarchy?

**Comment by diablo1281** [comment]
Hi, any news about bringing this feature back?

**Comment by diablo1281** [comment]
We are approaching V5 and the same problem persists...
